imnodes.h

```diff
@@ -115,10 +123,13 @@ void EndNode();
 // for output attributes.
 //
 // The attribute ids must be unique.
-void BeginInputAttribute(int id);
-void BeginOutputAttribute(int id);
+void BeginInputAttribute(int id, PinShape shape = PinShape::PinShape_Circle);
+void BeginOutputAttribute(int id, PinShape shape = PinShape::PinShape_Circle);
 void EndAttribute();
 
+// To get the link status for attributes.
+bool IsAttributeLinked(int id);
```

Comment:
Are you using this function in your node editor? I'm wondering if this could be an internal function for now, so that the `imnodes.h` wouldn't grow unless really needed 🙂

Reply:
It will be removed.